src/pages/Send/components/RecipientPicker/QuickSendItem/QuickSendItem.js

```diff
@@ -0,0 +1,64 @@
+// @flow strict
+
+import React, { PureComponent } from 'react'
+import classNames from 'classnames'
+import { t } from 'ttag'
+
+import { JPickerListItem } from 'components/base/JPicker'
+import { checkAddressValid } from 'utils/address'
+
+import itemStyles from './quickSendItem.m.scss'
+
+type Props = {|
+  +address: string,
+  +onChange: (address: string) => any,
+|}
+
+type ComponenState = {|
+  isFocused: boolean,
+|}
+
+export class QuickSendItem extends PureComponent<Props, ComponenState> {
+  state = {
+    isFocused: false,
+  }
+
+  handleFocus = () => {
+    this.setState({ isFocused: true })
+  }
+
+  handleBlur = () => {
+    this.setState({ isFocused: false })
+  }
```

Comment:
it could be simplified with https://github.com/jibrelnetwork/jwallet-web/blob/develop/src/utils/hooks/useFocus.js